src/table/pagination-table/components/head/TableHeadWrapper.tsx

```diff
@@ -80,7 +85,7 @@ const TableHeadWrapper = () => {
               onKeyDown={handleKeyDown}
               onMouseDown={handleMouseDown}
             >
-              <HeadCellContent column={column} isActive={isActive} isInteractionEnabled={isInteractionEnabled}>
+              <HeadCellContent column={column} isSorted={isSorted} isInteractionEnabled={isInteractionEnabled}>
```

Comment:
I find sorted a bit misleading, all columns are sorted in some way, but not always perfectly ascending/ascending (i.e. index first in `qEffectiveInterColumnSortOrder`). I think `isActivelySorted` was a good name for it